src/main/java/org/folio/des/scheduling/BursarExportScheduler.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.des.scheduling;
+
+import org.folio.des.domain.dto.ExportConfig;
+import org.folio.des.scheduling.quartz.QuartzExportJobScheduler;
+import org.folio.des.scheduling.quartz.converter.bursar.ExportConfigToBursarJobDetailConverter;
+import org.folio.des.scheduling.quartz.converter.bursar.ExportConfigToBursarTriggerConverter;
+import org.folio.des.scheduling.quartz.job.bursar.BursarJobKeyResolver;
+import org.quartz.Scheduler;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class BursarExportScheduler {
+
+
+  private final ExportConfigToBursarTriggerConverter exportConfigToBursarTriggerConverter;
+  private final BursarJobKeyResolver bursarJobKeyResolver;
+  private final ExportConfigToBursarJobDetailConverter exportConfigToBursarJobDetailConverter;
+  private final Scheduler scheduler;
+
+
+  public void scheduleBursarJob(ExportConfig exportConfig) {
+    if (exportConfig == null) {
+      log.error("exportConfig is null");
```

Comment:
https://wiki.folio.org/display/DD/Logging according to logging approach it should be a warn and also method name should be added, like this
log.warn("scheduleBursarJob:: exportConfig is null")